modelhub/tests_modelhub/data_and_utils/utils.py

```diff
@@ -81,7 +81,7 @@ def get_parsed_objectiv_data(engine):
         value = json.loads(value)
         # BQ uses time from taxonomy json for getting moment and day
```

Comment:
comment is outdated. Suggestion is not great ;)
```suggestion
        # BQ, Athena use time from taxonomy json for getting moment and day
```

Reply:
Fixed https://github.com/objectiv/objectiv-analytics/pull/1251/commits/4c500d65b517e2bc594afd849f773510a66cc646